src/components/v5/common/Fields/RadioButtons/ButtonRadioButtons/ButtonRadioButtons.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FC } from 'react';
+import clsx from 'clsx';
+import Icon from '~shared/Icon';
+import RadioButtonsBase from '../RadioButtonsBase';
+import { ButtonRadioButtonsProps } from './types';
+import { RadioItem } from '../RadioButtonsBase/types';
+
+const ButtonRadioButtons: FC<ButtonRadioButtonsProps> = ({
+  items,
+  className,
+  ...rest
+}) => {
+  const modifiedItems = items.map<RadioItem<string>>(
+    ({ label, iconName, colorClassName, ...item }) => ({
+      ...item,
+      // eslint-disable-next-line react/no-unstable-nested-components
+      children: ({ checked }) => (
+        <span
+          className={clsx(
+            colorClassName,
+            `
+              flex
+              items-center
+              justify-center
+              gap-1.5
+              border
+              border-current
+              transition
+              py-2
+              px-3
+              w-full
+              min-h-[2.5rem]
+              rounded-lg
+            `,
+            {
+              'bg-current': checked,
+            },
+          )}
+        >
+          {iconName && (
+            <Icon
+              className={clsx('h-[1em] w-[1em] text-[1.125rem]', {
```

Comment:
why do you use `em` here?

Reply:
That's one of the ways to scale the icon proportionally. I'm using ems to set width and height and then I'm applying font size on it to make it the right size. This way I only have to check the height of the icon and it will set the width accordingly.

Reply:
ok